pkg/agent/debugspy/placeholder.go

```diff
@@ -1,3 +1 @@
-// +build !debugspy
```

Comment:
It seems the build tags in empty "placeholders" are redundant.

Reply:
Not exactly... I think I added those because otherwise imports here don't work: https://github.com/pyroscope-io/pyroscope/blob/main/pkg/agent/session.go#L14-L17

Maybe we should add a comment in this case because I this is not intuitive

Reply:
Oh, I think I misunderstood your comment. I see what you're saying. Yeah, we can probably drop them in some of these files.